HW2_Laskina/calculator.py

```diff
@@ -0,0 +1,63 @@
+correct_oper = ["+", "-", "*", "/"]
+def div(x: float, y: float) -> float:
+    if y == 0:
+        raise ZeroDivisionError("Division by zero is not allowed")
+    return x / y
+
+def add(x: float, y: float) -> float:
+    return x + y
+  
+def mul(x: float, y: float) -> float:
+    return x * y
+
+def sub(x: float, y: float) -> float:
+    return x - y
+
+def parse_number(token: str):
+    if "." in token or "e" in token.lower():
+        return float(token)
+    return int(token)
+def main():
```

Comment:
```suggestion
    return int(token)
    
def main():
```

Не хватает отступов в паре мест. Это очень помогает читаемости, честное слово.